src/controller/python/chip-device-ctrl.py

```diff
@@ -762,6 +762,40 @@ def resolve(fabric_id: int, node_id: int) -> Dict[str, Any]:
     except Exception as e:
         return __get_response_dict(status = StatusCodeEnum.FAILED, error = str(e))
 
+def zcl_command(cluster: str,command: str, node_id: int, endpoint_id: Optional[int] = 1, group_id: Optional[int] = 0, args: Optional[dict] = {}) -> Dict[str, Any] :
+    try:
+        __check_supported_os()
+
+        all_commands = device_manager.devCtrl.ZCLCommandList()
+        command_arg = all_commands.get(cluster).get(command, None)
+        if command_arg is None:
+            raise exceptions.UnknownCommand(cluster, command)
+        commandArgs = __format_zcl_arguments_from_dict(args, command_arg)
+
+        err, res = device_manager.devCtrl.ZCLSend(cluster, command, node_id, endpoint_id, group_id, commandArgs, blocking=True)
+        if err != 0:
+            return __get_response_dict(status = StatusCodeEnum.FAILED)
+        elif res != None:
```

Comment:
can simply be `elif res:` and in the future use `is not` instead of `!=` it is more pythonic.

Reply:
Also I was just researching my comment about using `is not` in the future.... please disregard that comment per:

```
"The == operator compares the value or equality of two objects, whereas the Python is operator checks whether two variables point to the same object in memory. In the vast majority of cases, this means you should use the equality operators == and !=, except when you’re comparing to None."
```

from https://realpython.com/python-is-identity-vs-equality/

Reply:
Ack